docs/src/main/asciidoc/security-jdbc.adoc

```diff
@@ -167,6 +167,8 @@ quarkus.datasource.jdbc.url=jdbc:postgresql:elytron-security-jdbc
 ----
 
 In our context, we are using PostgreSQL as identity store, and we initialize the database with users and roles.
+We will use the salted and hashed version of `password` as a password in this example.
+We can use the `BcryptUtil` class to generate passwords in the Modular Crypt Format (MCF).
```

Comment:
@wowselim Thanks, can you please add some basic code example, showing a BcryptUtil package import and how to use it.

Reply:
Done in [e3f2004](https://github.com/quarkusio/quarkus/pull/39122/commits/e3f20046db9750165b33ec181ca65e1ff7b60b2c)